src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -43,6 +44,7 @@ public sealed class DashboardWebApplication : IAsyncDisposable
     private readonly IReadOnlyList<string> _validationFailures;
     private Func<EndpointInfo>? _frontendEndPointAccessor;
     private Func<EndpointInfo>? _otlpServiceEndPointAccessor;
```

Comment:
Can you change:

* _otlpServiceEndPointAccessor -> _otlpServiceGrpcEndPointAccessor
* OtlpServiceEndPointAccessor -> OtlpServiceGrpcEndPointAccessor

If there are two OTLP endpoint types then the variables should always say whether it's the grpc or http.